cypress/integration/form.js

```diff
@@ -75,4 +78,46 @@ context('Form Blocks Tests', () => {
     cy.get('#page-document button.submit-block-button').click();
   });
 
+  it('As editor I can add a multi-line text field to a form', () => {
+    cy.get('.block.inner.text .public-DraftEditor-content').click();
+    cy.get('button.block-add-button').click();
+    cy.findByText('Form').click();
+    cy.findByText('Textarea').click();
+    cy.get('input[id=field-textarea-label]').type('My name');
+    cy.get('#toolbar-save').click();
+    cy.url().should('eq', Cypress.config().baseUrl + '/my-page');
+    cy.findByText('My name').should('exist');
```

Comment:
@iFlameing please always add a blank line to separate the action (add block) from the test assertion (added block exists)